### DISCUSSION TITLE
OOD model applicability

### DISCUSSION THREAD

**Discussion by cschwarz68** [continuousml/Awesome-Out-Of-Distribution-Detection#9] [Q&A]
Thank you for making this excellent resource!
Both of the libraries/frameworks here are for PyTorch. I'd like to know if OOD has been applied to other types of models and if the ideas carry over fine? If I wanted to integrate OOD into my Random Forest model, for example, is there a subset of the papers that would be perfect for me to focus on?
Thanks much,
Chris

**Comment by huytransformer** [comment]
Hi Chris,

Thank you for your interest. My understanding is that **OOD Detection** is a term that has mostly caught on since the seminal paper by Hendrycks et al. (2017), and mostly in the context of deep learning. As you may see in some of the survey papers, the underlying concept existed long before that and evolved through different names in slightly different setups: anomaly detection, novelty detection, open set recognition, outlier detection, etc. It is almost certain that you would be able to find a method suitable for your setup if you search for those keywords, either in this repo, in the surveys, or through a search engine. I hope this gives you a useful pointer :).

Cheers
Huy
